docs/docs/html-generation.md

```diff
@@ -95,7 +95,7 @@ Finally, we call [react-dom](https://reactjs.org/docs/react-dom.html) and render
 
 ## build-html.js
 
-So, we've built the means to generate HTML for a page. This webpack bundle is saved to `public/render-page.js`. Next, we need to use it to generate HTML for all the site's pages.
+So, we've built the means to generate HTML for a page. These webpack bundles are saved to `.cache/_routes/`. Next, we need to use it to generate HTML for all the site's pages.
```

Comment:
Just syncinc docs changes with code changes.

Tho the question is should we already make this doc change now as it won't be in `latest` for some time still

```suggestion
So, we've built the means to generate HTML for a page. These webpack bundles are saved to `.cache/page-ssr/routes`. Next, we need to use it to generate HTML for all the site's pages.
```